app-editors/vscode/vscode-1.69.2.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop pax-utils xdg
+
+DESCRIPTION="Multiplatform Visual Studio Code from Microsoft"
+HOMEPAGE="https://code.visualstudio.com"
+SRC_URI="
+	amd64? ( https://update.code.visualstudio.com/${PV}/linux-x64/stable -> ${P}-amd64.tar.gz )
+	arm? ( https://update.code.visualstudio.com/${PV}/linux-armhf/stable -> ${P}-arm.tar.gz )
+	arm64? ( https://update.code.visualstudio.com/${PV}/linux-arm64/stable -> ${P}-arm64.tar.gz )
+"
+S="${WORKDIR}"
+
+RESTRICT="mirror strip bindist"
```

Comment:
Why mirror restrict? I skimmed over the license file and couldn't see anything that restricts it. + The distfile is hosted in Github too...

Reply:
I probably did it because I supposed Microsoft didn't like people redistributing their packages. But yeah it looks like they are not prohibiting that in their license. I will change that for my next bump.

Reply:
@ran-dall If you'd like, I can fix it on this PR, np.

Reply:
I'm about to commit soon so I'll do it to take full responsibility if something goes south :)

Reply:
```
app-editors/vscode
  MissingLicenseRestricts: version 1.69.2: EULA license 'Microsoft-vscode' requires RESTRICT="mirror"
```
I have absolutely no idea how pkgcheck figures that out, but reading the license with a looking glass I guess this is the part:

> 3.  **UPDATES.** The software may periodically check for updates and
>    download and install them for you. You may obtain updates only from
>    Microsoft or authorized sources. 

But then again it's talking about updates, not first-time installation ;) oh well maybe I'll leave it on vscode for now.